core/commands/add.go

```diff
@@ -258,7 +258,7 @@ You can now check what blocks have been created by:
 			exch = offline.Exchange(addblockstore)
 		}
 
-		bserv := blockservice.New(addblockstore, exch)
+		bserv := blockservice.New(addblockstore, exch) // hash security 001
```

Comment:
Does this need the hash security comment?

Reply:
This is the only place `n.Exchnage` is used standalone.